assets/css/challenge2.css

```diff
@@ -96,28 +96,29 @@ body {
   position: absolute;
   margin-top: 300px;
   margin-left: 300px;
- 
+
 }
 
 #davids_spaceship {
   width: 100px;
   position: absolute;
   margin-top: 500px;
   margin-left: 50px;
-  
+
 
 }
 
 #biscuit_ufo {
   position: absolute;
   width: 300px;
   top: -30px;
-  
-  
+
```

Comment:
Make sure you're only using single spaces & using spaces to separate style blocks